### ISSUE TITLE
Review Suggestion: Minor paper edits

### ISSUE THREAD

**Issue by shivupa** [flatironinstitute/solid_dmft#23]
### Summary
In the summary section, I think citing other software that are grey-box tools that allow for quick prototyping of ideas/flexible implementations would be helful. Some suggestions off the top of my head
```
iTensor - DMRG/tensor networks https://itensor.org/
INQ - DFT/TDDFT https://arxiv.org/abs/2106.03872
Quantum Package - correlated wave function methods https://quantumpackage.github.io/qp2/
PySCF - gaussian based wave function methods https://pyscf.org/
```
Each of these let you override the default implementations of algorithms if desired. These are just suggestions so if you disagree/feel that other packages would be a better fit feel free to replace.

### Design Principles
lines 35-36:
This allows **the user** to easily run ab-initio calculations for strongly correlated materials, as well as implement and test new features of TRIQS and benchmark new solvers against existing ones.

This reads awkwardly without the above addition ("the user").

**Comment by merkelm** [comment]
Thank you for your review! We addressed these points with pull request #26 on the `joss` branch (together with the remarks to the paper in issue #24). We ended up citing your suggestion of pySCF together with the ASE as a reference for gray-boxing approaches in our field and also fixed the "allows to" formulation.

**Comment by shivupa** [comment]
Thanks this addressed my concerns!
